docs/source/tutorials/pipeline/1_basics.py

```diff
@@ -0,0 +1 @@
+/home/avsakharov/deeppavlov/dialog_flow_framework/tutorials/pipeline/1_basics.py
```

Comment:
what is it ? This one absolute path exists only for avsakharov

Reply:
Deleted